projects/sdao/index.js

```diff
@@ -0,0 +1,15 @@
+const fetchTvl = async function () {
+  const response = await fetch(
+    "https://api.singularitydao.ai/value/totalValueLocked"
```

Comment:
Can this be calculated using on-chain calls instead? We've got a guide on how to use our SDK here: https://docs.llama.fi/list-your-project/submit-a-project